site/en/docs/privacy-sandbox/topics-experiment/index.md

```diff
@@ -19,17 +21,6 @@ authors:
 
 ## Try the API
 
-{% Aside 'caution' %}
-
-Not all users may be eligible for the Privacy Sandbox Relevance and Measurement origin trial, even
-on pages that provide a valid trial token.
-
-[Testing the Privacy Sandbox ads relevance and measurement APIs](/blog/privacy-sandbox-unified-origin-trial#eligible-users)
-explains why this is so, and shows how you can (and should) detect if an origin trial feature is
-available before attempting to use it.
-
-{% endAside %}
-
 1. Check the Privacy Sandbox [status page](/docs/privacy-sandbox/status/#topics) for updates on the
 implementation status of the Topics API.
 2. Experiment with the API:
```

Comment:
The "Learn how to take part ..." bullet can go as well.

Reply:
We should probably also update "Use the Topics JavaScript API to access topics ..." to suggest using either the header approach (fetch or iframe) or the JavaScript API.